public/documents/fari-rpgs/charge-rpg.md

```diff
@@ -2334,6 +2334,300 @@ The original idea comes from [Ironsworn](https://ironswornrpg.com)'s vows. `Goal
+
+
+### Party Danger and Progress
+
+When it makes sense as a `consequence` of events in the session, the GM may mark the `danger` clock on the `party sheet`. This clock tracks a generally deteriorating position in society - loss of assets, adversaries growing more hostile, or alliances shifting. In a similar fashion, the `progress` clock is available for when significant events happen in a session that might improve the party’s position. 
+
+When these clocks fill, they are reset and a `supporting detail` changes. If the `danger` clock fills, this change is generally for the worse. If the `progress` clock fills, the change is generally for the better. 
+
+### Burning Progress
+
+Sometimes a `party` will have to sacrifice their long term goals to focus on the short term goals. If it fits the narrative, `progress` can be burned as `momentum` to represent this on a one-for-one basis.
+
+## Concepts and Details
+
+> #### Designer Note
+> 
+> Key Concepts and Supporting details can be a way for game designers to help drive the kinds of stories that make sense in their setting. The categories provided below are intended to be a generic starting point that can be made setting specific - renaming them per setting is an easy customization to set the right mood!
+> 
+> Further structure can be added to the supporting details to drive home key points in the setting, for example, asking players to choose a specific adversary and ally from a list of available world forces.
+> 
+> Starting positions for these key details can help further drive a story for the group of players to build on. A crew looking for their first big score might not have a lot of assets or allies, but they probably aren’t big enough to be on anyone’s radar as an adversary either. A resistance group against a repressive regime might start in a horrible position against authority, but have a larger list of allies to lean on. 
+
+Like with character creation, `details` on the `party sheet` can offer both fictional support (helping build on the story) and narrative liberties (giving players permission to take certain actions). While the GM will have the final say, the entire party should have a chance to help develop and decide on these details at character creation. There will also have the chance to change these over time as the `party` advances.  
+
+### Key Concepts
+
+Each `key concept` focuses on the position of the `party` along an important axis. The `disposition` of the party in these categories may be represented as the sum of positive `supporting details`, subtracting any negative `supporting details`, resulting in a range from -4 to +4. 
+
+`Supporting details` are specific people, places, things or ideas related to the party in some sort of way, while `key concepts` are overarching, category level descriptions of the party’s standing. 
+
+* **Assets** - This represents the available resources of the party, including money and any available equipment.
```

Comment:
suggestion: perhaps we should say something like "There are 4 `key concepts`:" and then list them?

Reply:
Agree